### ISSUE TITLE
Explore the ability to do comparisons 

### ISSUE THREAD

**Issue by jrgwhiteford** [RobJN/regional-fes#29]
ability to do side-by-side comparisons has been discussed - this could be scenarios / parameters / region v region / region v GB wide

**Comment by slowe** [comment]
I'm imagining some kind of side-by-side selectors like this:
![Screenshot 2022-05-11 at 18-52-16 FES 2021 Compare](https://user-images.githubusercontent.com/299787/167914655-450fae87-22a4-47cf-98df-8575b65d3cf1.png)
With the comparisons below.

**Comment by slowe** [comment]
@jrgwhiteford Yes. It would be on a separate visualisation page. The two selections (each consisting of scenario+parameter+area) would be in side-by-side columns. Is requiring scenario+parameter+area OK?

Below the dropdown boxes, in each column, would be the maps. I think the maps will help people know where a region is but will probably be of limited use to understand the data because, if the user selects "East of England", they'd just get the East of England polygon coloured in (using the colour scale for that scenario+parameter across the full year range?). Given that a heat map can only show one year at a time, we would probably then want year sliders for each map so that specific years could be chosen. As in my example dropdowns above, we could let the area selector include multiple regions e.g. "GB (regions)" which would then show the polygons for each of the GB regions for the chosen year (also with a colour scale set to the full year range). 

Below the maps could be line charts with time along the x-axis. We could potentially highlight the year that is selected in the year sliders (above). If that column includes multiple regions (e.g. "GB (regions)") we could plot multiple series on the graph.

To make it easier to compare when things happen in the data, we could potentially also have a column-spanning line chart with both selections plotted together.

I think this lets a user:
* select the same scenario with the same parameter and compare different years
* select the same parameter in different scenarios
* compare different scenario+parameters over time
* see a line chart showing how a scenario+parameter varies between areas

Does this give the flexibility that you want in comparing things?

**Comment by slowe** [comment]
@jrgwhiteford At the moment I have an open pull request about style changes and a download button for the main visualisation see #36. [Our Github pages branch is currently showing that](https://open-innovations.github.io/regional-fes/2021-FES/electricity-maps.html) so that you can review it before accepting that pull request. Once you've had chance to look at that I can switch over the Github pages to show the branch I'm working on for this issue.

**Comment by jrgwhiteford** [comment]
Hi Stuart i have merged the most recent changes

**Comment by slowe** [comment]
@jrgwhiteford Thanks. I've switched the Open Innovations GitHub pages branch so you can [play with the comparison tool](https://open-innovations.github.io/regional-fes/2021-FES/compare) as it is now.

**Comment by jrgwhiteford** [comment]
Thanks @slowe I think having the colours matching the different regions makes sense. I like the ability to download as SVG. Is this feature possible on the main visualisation? is there an option on the main visualisation to remove place names (as a layer?)

**Comment by slowe** [comment]
The maps on the main vis have pixel-based map tile backgrounds which don't lend themselves to SVG. The polygons would be doable. Would it be confusing for people to only get part of the view?

Yes, we could make the label layer togglable.

**Comment by jrgwhiteford** [comment]
Yes i think you are right that this would be confusing for people
